README.md

```diff
@@ -123,6 +123,67 @@ Every target has a clean target, so in order to clean swss, execute:
 
 It is recommended to use clean targets to clean all packages that are built together, like dev packages for instance. In order to be more familiar with build process and make some changes to it, it is recommended to read this short [Documentation](README.buildsystem.md).
 
+## Build debug dockers and debug SONiC installer image:
```

Comment:
in my original comments, I was refering to README.buildimage.md

Can you put how to build debug image information. You can add the hyper link in this page, but I prefer not to talk about the debug image in detail in this md file. I'd like to keep it simpler.

Reply:
in the buildimage md file, you may also want to explain extra build target you have added so that user knows how to add dbg symbols for their new packages introduced to sonic.